packages/experiments/src/components/Microfeedback/Microfeedback.base.tsx

```diff
@@ -0,0 +1,145 @@
+    const dislikeIcon = this.state.vote === 'dislike' ? 'DislikeSolid' : 'Dislike';
+    const hideThumbsDownCallout = this.state.vote !== 'dislike' || !this.state.isFollowupVisible;
+    const hideThumbsUpCallout = this.state.vote !== 'like' || !this.state.isFollowupVisible;
+    const onCalloutDismiss = this._onCalloutDismiss.bind(this);
+
+    const classNames = getClassNames(this.props.styles, {
+      theme: this.props.theme
+    });
+
+    return (
+      <div className={classNames.root}>
+        <Stack horizontal styles={microfeedbackStyles}>
+          <div ref={this.likeRef}>
+            <IconButton menuIconProps={{ iconName: likeIcon }} title={this.props.thumbsUpTitle} onClick={this._vote.bind(this, 'like')} />
+          </div>
+          <div ref={this.dislikeRef}>
+            <IconButton
+              menuIconProps={{ iconName: dislikeIcon }}
+              title={this.props.thumbsDownTitle}
+              onClick={this._vote.bind(this, 'dislike')}
+            />
+          </div>
+        </Stack>
+        {this.props.ThumbsUpQuestion ? (
+          <Callout
+            hidden={hideThumbsUpCallout}
+            role="alertdialog"
+            gapSpace={0}
+            target={this.likeRef.current}
+            setInitialFocus={true}
+            onDismiss={onCalloutDismiss}
+          >
+            <div>
+              <FocusZone direction={FocusZoneDirection.vertical}>
+                <Stack padding={10}>
+                  <Text variant="small">{this.props.ThumbsUpQuestion.question}</Text>
+                </Stack>
+                <List items={this.props.ThumbsUpQuestion.options} onRenderCell={this._onRenderCalloutItem} />
+              </FocusZone>
+            </div>
+          </Callout>
+        ) : null}
+        {this.props.ThumbsDownQuestion ? (
+          <Callout
+            hidden={hideThumbsDownCallout}
+            role="alertdialog"
+            gapSpace={0}
+            target={this.dislikeRef.current}
+            setInitialFocus={true}
+            onDismiss={onCalloutDismiss}
+          >
+            <FocusZone direction={FocusZoneDirection.vertical}>
+              <Stack padding={10}>
+                <Text variant="small">{this.props.ThumbsDownQuestion.question}</Text>
+              </Stack>
+              <List items={this.props.ThumbsDownQuestion.options} onRenderCell={this._onRenderCalloutItem} />
+            </FocusZone>
+          </Callout>
+        ) : null}
+      </div>
+    );
+  }
+
+  private _onCalloutDismiss(): void {
+    this.setState({ isFollowupVisible: false });
+  }
+
+  private _onRenderCalloutItem = (item: string, index: number | undefined): JSX.Element => {
+    return (
+      <DefaultButton data-is-focusable={true} styles={microfeedbackItemStyles} onClick={this._listOptions.bind(this, index)}>
+        <Text variant="small">{`${item}`}</Text>
+      </DefaultButton>
+    );
+  };
+
+  private _listOptions(option: number): void {
```

Comment:
Same here as comment above for `_listOptions` and below for `_vote`. It turns out `_onRenderCalloutItem` is already written in a way it doesn't need binding. This should allow you to remove all of the `bind` calls.

Reply:
Fixed